keylime/db/keylime_db.py

```diff
@@ -82,6 +82,7 @@ def make_engine(self, service: str) -> Engine:
             if not url.count("sqlite:"):
                 engine_args["pool_size"] = int(p_sz)
                 engine_args["max_overflow"] = int(m_ovfl)
+                engine_args["pool_pre_ping"] = True
```

Comment:
Did you saw this issue?

In any case this see the less intrusive change, the other one seems to be setting the pool_recycle parameter and managing the exception per query

Reply:
Yes, we saw it on our test infrastructure happening. You see then errors like this:
````
sqlalchemy.exc.OperationalError: (pymysql.err.OperationalError) (2006, "MySQL server has gone away (ConnectionResetError(104, 'Connection reset by peer'))")
````

Reply:
This is a good idea. We have been facing the same troubles in our long-running large-scale testing. It will not solve all the problems*, but I am going to test it out just to make sure it does impose too much "additional overhead" on the DB connections (don't expect it does).

* (I believe @aplanas is right and we will need to harden our exception catching and - more crucially - recovery per query)